packages/turf-transform-rotate/index.js

```diff
@@ -0,0 +1,114 @@
+var helpers = require('@turf/helpers');
+var point = helpers.point;
+var multiPoint = helpers.multiPoint;
+var lineString = helpers.lineString;
+var multiLineString = helpers.multiLineString;
+var polygon = helpers.polygon;
+var multiPolygon = helpers.multiPolygon;
+var feature = helpers.feature;
+var invariant = require('@turf/invariant');
+var getCoords = invariant.getCoords;
+var centroid = require('@turf/centroid');
+var rhumbBearing = require('@turf/rhumb-bearing');
+var rhumbDestination = require('@turf/rhumb-destination');
+var rhumbDistance = require('@turf/rhumb-distance');
+
+/**
+ * Rotates any geojson Feature or Geometry of a specified angle, around its `centroid` or a given `pivot` point;
+ * all rotations follow the right-hand rule: https://en.wikipedia.org/wiki/Right-hand_rule
+ *
+ * @name rotate
+ * @param {Geometry|Feature<any>} geojson object to be rotated
+ * @param {number} angle of rotation (along the vertical axis), from North in decimal degrees, negative clockwise
+ * @param {Geometry|Feature<Point>|Array<number>} [pivot=`centroid`] point around which the rotation will be performed
+ * @param {number} [xRotation=0] angle of rotation, respect to the horizontal (x,y) plain, about the x axis;
+ * @param {number} [yRotation=0] angle of rotation, respect to the horizontal (x,y) plain, about the y axis;
+ * @returns {Feature<any>} the rotated GeoJSON feature
+ * @example
+ * var poly = turf.polygon([[0,29],[3.5,29],[2.5,32],[0,29]]);
```

Comment:
We should add these tests directly in the main Turf tests [`package/turf/test.js`](https://github.com/Turfjs/turf/blob/master/packages/turf/test.js).

A test that actually tests if all the JSDocs are being tested, it's currently doing this in `turf-www`.

@rowanwins